Image Viewing GUI/main.py

```diff
@@ -0,0 +1,82 @@
+# importing the modules
+
+from tkinter import *
+from PIL import ImageTk, Image
+
+root = Tk()
+root.title('Image viewer')
+root.resizable(0, 0)
+
+# opening the images from the image folder
+img1 = Image.open("./Image Viewing GUI/images/img1.jpg")
```

Comment:
Implement this suggestion by @antrikshmisri  https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/751#discussion_r602692725 instead of hardcoding the images loading